### DISCUSSION TITLE
Admin Hub UI Design Improvements

### DISCUSSION THREAD

**Discussion by glennjacobs** [lunarphp/lunar#1043] [Feature Requests]
Currently, the admin hub is very much based upon Tailwind UI basic components, which worked well for the period of rapid development. However, the fundamental functionality is now starting to stabilise, so we wish to improve the hub's UI.

Things we want to achieve are...

- Consistency 
- More efficient use of space
- UI components which are designed to be extended
- Professional design
- Subtle UI animations to support the UX

**Comment by AidasK** [comment]
I think you are trying to reinvent a wheel here. Since you are an opensource project I would suggest you guys to use https://github.com/filamentphp/filament/ instead. It already covers all your requirements and even more. Lunar admin panel is never going to be as good as filamentphp one so I suggest you guys to reevaluate your roadmap and focus on ecommerce part instead.

**Comment by glennjacobs** [comment]
> I think you are trying to reinvent a wheel here. Since you are an opensource project I would suggest you guys to use https://github.com/filamentphp/filament/ instead. It already covers all your requirements and even more. Lunar admin panel is never going to be as good as filamentphp one so I suggest you guys to reevaluate your roadmap and focus on ecommerce part instead.

We have reviewed FilamentPHP and we do not wish to use it. I understand it is very good for CRUD applications, but we wish to maintain full control over our UI.

**Comment by mustafa-online** [comment]
Please add "**RTL Support**" to this list.
